scripts/jenkins/qat/Makefile

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Makefile for QAT-DPDK jenkins.
+
+all: checks plugin tests
+
+checks:
+	bash checks.sh
```

Comment:
`checks` has the problem that it assumes this is run on a node that is QAT ready. that may not be the case always. I think it'd be better if you just deploy the plugin and return the error messages.

Reply:
@mythi Do you think we should only skip `checks` target from `all` and preserve scripts to execute it separately "just in case", or we should remove both script and rule/target?

Reply:
I think we could just drop both. If we find the device plugin start-up checks misses some system pre-condition, we could look into inserting an init-container to run those checks.. but it's not necessary right now IMO.